code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+const ThoughtForm = ({onFormSubmit, setNewThought, newThought}) => {
+
+    return (
+
+      <form onSubmit={onFormSubmit} className="form-container">
+        <label htmlFor="newThought" className="label-form">What's making you Happy right now?</label>
+        <textarea 
+          className="thought-input"
+          id="newThought"
+          type="text" 
+          value={newThought}
+          onChange= {(e) => setNewThought(e.target.value)}
+          ></textarea>
+          <div className="button-count">
+            <button disabled = {newThought.length<5 || newThought.length>140}type="submit" className="submit-button">
+              <div className="send-wrapper">
+                <img src="/assets/red-heart.png" alt="red heart" className="heart"></img> 
```

Comment:
you can tell you've really thought about accessibility. Super!(as Maks would say)